src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -377,12 +395,13 @@ public AdsBaseObject waitForPageLoaded() {
   }
 
   public AdsBaseObject waitForPageLoadedWithGpt() {
+    timerStart();
     waitForPageLoaded();
 
     String waitForGPTJS = "typeof window.googletag === 'object'";
     jsActions.waitForJavaScriptTruthy(waitForGPTJS);
-    PageObjectLogging.log("GPT Loaded", String.valueOf(jsActions.execute(waitForGPTJS)), true);
-
+    PageObjectLogging.log("GPT Loaded", "after " +
+                                        String.valueOf(getNumberOfSecondsFromStart()) + " s", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Directly append the argument of String.valueOf(). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1153)